pkg/patterns/addon/pkg/status/aggregate.go

```diff
@@ -45,12 +45,16 @@ type aggregator struct {
 	client client.Client
 }
 
-func (a *aggregator) Reconciled(ctx context.Context, src declarative.DeclarativeObject, objs *manifest.Objects, _ error) error {
+func (a *aggregator) Reconciled(ctx context.Context, src declarative.DeclarativeObject, objs *manifest.Objects, reconcileErr error) error {
 	log := log.FromContext(ctx)
 
 	statusHealthy := true
 	statusErrors := []string{}
 
+	if reconcileErr != nil {
+		statusHealthy = false
```

Comment:
Do you also want to propagate the message to the user? It's a little blurry here as its sort of internal, but it would aid in debugging.

If so add:
statusErrors = append(statusErrors, fmt.Sprintf("%v", reconcileErr))